### PR TITLE
Update CHANGELOG.md and README.md for preview4

### PR DESCRIPTION
**Pull request by sarkar-rajarshi**

### CODE REVIEW

File: sdk/communication/azure-communication-chat/CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,10 @@
 
 ## 1.0.0b5 (2021-03-09)
 - Updated `azure-communication-chat` version.
```
Comment (ankitarorabit) [inline sdk/communication/azure-communication-chat/CHANGELOG.md:4]: You can get rid of this as it was added since there was no content

**Review by ankitarorabit — APPROVED** [review]

File: sdk/communication/azure-communication-chat/CHANGELOG.md [hunk 2]
```diff
@@ -1,7 +1,9 @@
 # Release History
 
 ## 1.0.0b5 (2021-03-09)
-- Updated `azure-communication-chat` version.
+- Added support for communication identifiers instead of raw strings.
```
Comment (ankitarorabit) [inline sdk/communication/azure-communication-chat/CHANGELOG.md:4]: Please confirm if they are breaking changes and if so include that sub-heading

Comment (sarkar-rajarshi) [reply sdk/communication/azure-communication-chat/CHANGELOG.md:5]: @ankitarorabit Done

Comment (ankitarorabit) [reply sdk/communication/azure-communication-chat/CHANGELOG.md:5]: Thank you!

**Pull request merged**